src/View/Includes/_footer.html.twig

```diff
@@ -0,0 +1,35 @@
+<div class="footer-dark">
+    <footer>
+        <div class="container">
+            <div class="row text-center">
+                <div class="col-sm-6 col-md-4 item">
+                    <h3>A propos de nous</h3>
```

Comment:
honnest question : are there h2s on this page before these h3s?

Reply:
he may have a h2 later